cdm/core/src/main/java/ucar/nc2/ft2/coverage/HorizCoordSys.java

```diff
@@ -355,39 +355,26 @@ public LatLonPoint getLatLon(int yindex, int xindex) {
    * cases:
    * A. wantMin < wantMax
    * 1 wantMin, wantMax > end : empty
-   * 2 wantMin, wantMax < end : [wantMin, wantMax]
-   * 3 wantMin < end, wantMax > end : [wantMin, end]
+   * 2 wantMin < end : [wantMin, min(wantMax,end)]
    * 
    * B. wantMin > wantMax
    * 1 wantMin, wantMax > end : all [start, end]
    * 2 wantMin, wantMax < end : 2 pieces: [wantMin, end] + [start, max]
-   * 3 wantMin < end, wantMax > end : [wantMin, end]
    */
   private List<MAMath.MinMax> subsetLonIntervals(double wantMin, double wantMax, double start, double end) {
```

Comment:
not really related to your changes, but do we need to verify for instance that start < end or 0 <= start, end < 360

Reply:
The comments _claim_ that it's all been normalized to start, and end = start + 360, so I'm just trusting that...for now